R/twas.R

```diff
@@ -315,8 +316,9 @@ twas_weights_cv <- function(X, Y, fold = NULL, sample_partitions = NULL, weight_
       # Remove columns with zero standard error
       valid_columns <- apply(X_train, 2, function(col) sd(col) != 0)
       X_train <- X_train[, valid_columns, drop = F]
-      X_train <- filter_X(X_train, imiss_cutoff, min_cv_maf, Y = Y_train)
-      valid_columns <- colnames(X_train)
+      # X_train <- filter_X(X_train, imiss_cutoff, min_cv_maf, Y = Y_train, drop_by_Y_only = TRUE)
```

Comment:
looks like you are no longer using imisscuttoff on each fold of data? i'm fine with it as long as you are fine with it. ...

Reply:
and if so, please remove these commented out lines to avoid confusion

Reply:
Yep, addressed in the new PR